.gitignore

```diff
@@ -1 +1,3 @@
 /.idea/
+.DS_Store
+.gitignore
```

Comment:
Why would you put the .gitignore file into the .gitignore file? Ignoreception

Reply:
Just my Git repo being stupid. Remove the .gitignore entry, das store makes sense though (created by Mac automatically)